tsconfig.json

```diff
@@ -28,6 +28,7 @@
       "ABIs/*": ["ABIs/*"],
       "assets/*": ["assets/*"],
       "components/*": ["components/*"],
+      "componentsWork/*": ["componentsWork/*"],
```

Comment:
would it be easier to just put the rebrand components into `/components/work` and then just remove the `work` directory once they replace the other ones?